lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -43,32 +44,67 @@ class FeatureCard extends React.Component {
   }
 
   componentDidMount() {
-    this.serverRequest = $.get(`${API_URL}campaigns?random=true&staff_pick=true`, result => {
-      // Filter out campaigns users have signed up for & then sort remaining campaigns
-      const userSignups = Drupal.settings.dsUser.activity.signups;
-      const campaigns = result.data.filter(campaign => userSignups.indexOf(parseInt(campaign.id)) == -1).sort(sortCampaigns);
-
+    // First load staff pick campaigns to quickly initialize the card
+    this.getCampaigns({staffPick: true, campaigns: []}, 1, campaigns => {
       this.setState({
         campaigns: campaigns,
         current_campaign_index: 0
       });
+
+      // Then backfill with all of the other campaigns
+      this.getCampaigns({staff_pick: 0, campaigns: campaigns}, 1, campaigns => {
+        this.setState({
+          campaigns: campaigns
+        });
+      });
     });
   }
 
   componentWillUnmount() {
-    if (this.serverRequest) {
-      this.serverRequest.abort();
+    if (this.requests) {
+      for (var request in this.requests) {
+        request.abort();
+      }
     }
   }
 
+  getCampaigns(data, page, cb) {
+    this.requests.push($.get(`${API_URL}campaigns?staff_pick=${data.staff_pick}&page=${page}`, result => {
+      let rawCampaigns = result.data;
+
+      // Filter out campaigns users have signed up for
+      const userSignups = Drupal.settings.dsUser.activity.signups;
+      rawCampaigns = rawCampaigns.filter(campaign => userSignups.indexOf(parseInt(campaign.id)) == -1);
+
+      // Remove any non staff picks
+      if (!data.staffPick) {
+        rawCampaigns = rawCampaigns.filter(campaign => campaign.staff_pick == false);
```

Comment:
Is this necessary since you're doing this above with the `?staff_pick=` query parameter?